dotCMS/src/main/java/com/dotmarketing/business/VersionableFactoryImpl.java

```diff
@@ -328,42 +339,67 @@ protected ContentletVersionInfo findContentletVersionInfoInDB(String identifier,
     }
     @Override
     protected void saveContentletVersionInfo(ContentletVersionInfo cvInfo, boolean updateVersionTS) throws DotDataException, DotStateException {
-    	Identifier ident = this.iapi.find(cvInfo.getIdentifier());
-    	ContentletVersionInfo vi= null;
-    	if(ident!=null && InodeUtils.isSet(ident.getId())){
-    		vi= findContentletVersionInfoInDB(ident.getId(), cvInfo.getLang());
-    	}
-    	boolean isNew = vi==null || !InodeUtils.isSet(vi.getIdentifier());
-        try {
-			BeanUtils.copyProperties(vi, cvInfo);
-		} catch (Exception e) {
-			throw new DotDataException(e.getMessage(),e);
+		boolean isNew = true;
+		if (UtilMethods.isSet(cvInfo.getIdentifier())) {
+			try {
+				final ContentletVersionInfo fromDB =
+						findContentletVersionInfoInDB(cvInfo.getIdentifier(), cvInfo.getLang());
+				if (fromDB != null) {
+					isNew = false;
+				}
+			} catch (final Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6214071)